pkg/query/api/v1.go

```diff
@@ -87,6 +87,26 @@ func (e *ApiError) Error() string {
 	return fmt.Sprintf("%s: %s", e.Typ, e.Err)
 }
 
+// ThanosVersion contains build information about Thanos.
+type ThanosVersion struct {
```

Comment:
Isn't this exactly equivalent to the struct in `pkg/ui/query.go`? Maybe let's re-use it?

Reply:
Yeah, sure.